api/notd/model.py

```diff
@@ -175,3 +175,40 @@ class TokenMultiOwnership(RetrievedTokenMultiOwnership):
     tokenMultiOwnershipId: int
     createdDate: datetime.datetime
     updatedDate: datetime.datetime
+
+@dataclasses.dataclass
+class RetrievedCollectionHourlyActivity:
+    address: str
+    date: datetime.datetime
+    transferCount: int
+    saleCount: int
+    totalVolume: int
+    minimumValue: int
+    maximumValue: int
+    averageValue: int
+
+@dataclasses.dataclass
+class CollectionHourlyActivity(RetrievedCollectionHourlyActivity):
+    collectionActivityId: int
+    createdDate: datetime.datetime
+    updatedDate: datetime.datetime
+
+@dataclasses.dataclass
+class CollectionStatistics:
+    date: datetime.date
+    itemCount: int
+    holderCount: int
+    totalTradeVolume: int
+    lowestSaleLast24Hours: int
+    highestSaleLast24Hours: int
+    tradeVolume24Hours: int
+
+@dataclasses.dataclass
+class CollectionActivity:
+    date: datetime.date
```

Comment:
why does this have a date? if anything it should have a start date and end date, or neither

Reply:
I thought we were plotting with date on the x-axis